site/content/integrate/apps/_index.md

```diff
@@ -1,33 +1,50 @@
 ---
 title: "Apps (Developers Preview)"
-heading: "Overview"
+heading: "Introduction"
 description: "Apps are lightweight interactive add-ons to Mattermost."
 section: "integrate"
 ---
 
-Apps are lighweight interactive add-ons to Mattermost. Apps can:
+Apps are lightweight, interactive add-ons to Mattermost which use serverless hosting to run without any dedicated infrastructure. Apps can:
 
 - Display interactive, dynamic Modal forms and Message Actions.
-- Attach themselves to locations in the Mattermost UI (e.g. channel bar buttons, post menu, channel menu, commands), and can add their custom /commands with full Autocomplete.
+- Be written in several different languages.
+- Attach themselves to locations in the Mattermost user interface (e.g. channel bar buttons, post menu, channel menu, commands), and can add their custom `/` commands with full Autocomplete.
 - Receive webhooks from Mattermost, and third-parties, and use the Mattermost REST APIs to post messages, etc.
-- Be hosted externally (HTTP), on Mattermost Cloud (AWS Lambda), and soon in self-managed as well as customers' own AWS environments.
-- Be developed in any language*.
+- Be hosted externally (HTTP) and on Mattermost Cloud (AWS Lambda)*.
+- Work on both Mobile and Desktop apps so developers can focus on the functionality of their apps.
+- Be deployed using our Mattermost serverless hosting infrastructure keeping data secure and supporting scalability.
 
 Apps are available as a Developers Preview and we're looking for your feedback! Share constructive feedback in the [Mattermost Apps channel](https://community.mattermost.com/core/channels/mattermost-apps) on our Mattermost community instance.
 
+* Apps will soon be available for self-managed deployments as well as customers' own AWS environments.
+
 Read the [quick start guide]({{< ref  "quick-start-go" >}}) to learn how to write your first app.
 
-# Contents
+## Mattermost serverless app hosting
+
+Mattermost serverless hosting allows for easy app installation from the App Marketplace by a System Admin and uses AWS Lambda serverless technology instead of relying on a physical server. Developers who create apps using a serverless development approach can easily deploy apps securely, efficiently, and at scale in the Mattermost Cloud.
+
+## Developing serverless apps
+
```

Comment:
This could do with some content: Would this be a place for a link to the best practices guide (not yet written) or maybe something about the apps framework and how it's related to development? @aaronrothschild?

Reply:
Agreed - @iomodo do you have any thoughts on what should go here?  

perhaps a blurb like:
Serverless technology allows for a "Fucntion" to lay dormant until it is needed - for example processing an incoming message from an external system (ie: a P1 incident in your tracking system) and sending a message to your Incident Response Channel in Mattermost.  This allows for reduced costs and less maintenance of persistent servers needed to host other applications.

Reply:
3/5 that hosting is not such an important topic that we need to talk about it upfront. It's a topic for developers, not for admins. All of the "hosting" docs should go into a separate page later. No need write about it now.

Reply:
I am 5/5 on adding this because these are the only docs linked in the hackathon spreadsheet and I think context is important. 

That said, I can cut this down to bullet lists to make it easier to scan?

Reply:
Is a link to the quick start guide enough?

Reply:
I moved some content around - LMK what you think.

Reply:
After the 16th of May there is no need for a local dev environment any longer. 2/5 that we shouldn't link to https://developers.mattermost.com/contribute/server/developer-setup/ right now